### ISSUE TITLE
Fastboot + ember-page-title with freshly generated app fails

### ISSUE THREAD

**Issue by iand675** [adopted-ember-addons/ember-page-title#77]
Steps to repro:

1. Generate an app running Ember 2.13.3 (I used the `--yarn` flag as well).
2. `ember install fastboot`
3. `ember install ember-page-title`
4. Remove the existing title tag from the index.html file
5. Run `ember serve` and hit the index page.
6. Get stack trace:

```
Error
    at assert (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:8031:1)
    at Object.assert (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:8244:1)
    at /home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:6717:1
    at Vertices.each (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:4775:1)
    at Vertices.topsort (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:4742:1)
    at DAG.topsort (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:4687:1)
    at Class._runInitializer (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:6733:1)
    at Class.runInstanceInitializers (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:6716:1)
    at Class._bootSync (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:4993:1)
    at /home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:6404:1
    at initializePromise (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:50230:1)
    at new Promise (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:50718:1)
    at Class.boot (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:6403:1)
    at buildAppInstance.then.appInstance (/home/ian/Code/super-hackage/node_modules/fastboot/src/ember-app.js:240:25)
    at tryCatch (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:50180:1)
    at invokeCallback (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:50195:1)
    at publish (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:50163:1)
    at /home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:40842:1
    at invokeWithOnError (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:2197:1)
    at Queue.flush (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:2256:1)
    at DeferredActionQueues.flush (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:2380:1)
    at Backburner.end (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:2450:1)
    at Timeout._onTimeout (/home/ian/Code/super-hackage/tmp/broccoli_merge_trees-output_path-jdMjWSLu.tmp/assets/vendor/ember/ember.debug.js:3016:1)
    at ontimeout (timers.js:386:14)
    at tryOnTimeout (timers.js:250:5)
    at Timer.listOnTimeout (timers.js:214:5)
```

**Comment by kratiahuja** [comment]
The will be resolved with https://github.com/ronco/ember-cli-head/pull/35 I guess

**Comment by sivakumar-kailasam** [comment]
This is now fixed with the latest ember-cli-head@0.3.1 which has the above PR merged which works with the latest version of ember-page-title.

**Comment by raido** [comment]
@knownasilya seems this is outdated and can be closed.
